index.html

```diff
@@ -1,176 +1,780 @@
+                    <label for="send_updates">Send me occasional email updates</label>
+                </div>
+
+                <div class="action_btns">
+                    <div class="one_half"><a href="#" class="btn back_btn"><i class="fa fa-angle-double-left"></i> Back</a></div>
+                    <div class="one_half last"><a href="#" class="btn btn_red">Register</a></div>
+                </div>
+            </form>
+        </div>
+    </section>
+</div>
 
-                }
+  <div class="main-banner wow fadeIn" id="top" data-wow-duration="1s" data-wow-delay="0.5s">
+    <div class="container" style="margin-bottom: 140px;">
+      <div class="row">
+        <div class="col-lg-12">
+          <div class="row">
+            <div class="col-lg-6 align-self-center">
+              <div class="left-content show-up header-text wow fadeInLeft" data-wow-duration="1s" data-wow-delay="1s">
+                <div class="row">
+                  <div class="col-lg-12">
+                    <h2>Login to foodprint</h2>
+                    <p>The recipe and diet planning app with full nutritional and ecological data. Available on Android, iOS and in browser! Register now and eat well with ease. Free forever, we promise!</p>
+                  </div>
+                  <!--
+                  <div class="col-lg-12">
+                    <div class="white-button first-button scroll-to-section">
+                      <a href="https://apps.apple.com/lt/app/cooking-fever/id714796093" target="_blank">App store <i class="fab fa-apple"></i></a>
+                    </div>
+                    <div class="white-button scroll-to-section">
+                      <a href="https://play.google.com/store/apps/details?id=com.nordcurrent.canteenhd" target="_blank"> Google Play<i class="fab fa-google-play"></i></a>
+                    </div>
+                  <br>
+                  -->
+                    <div class="white-button scroll-to-section">
+                      <a href="https://staging.quanteco.org/search" target="_blank"> Login Online</a>
+                    </div>
+                  </div>
+                </div>
+              </div>
+              <div class="col-lg-6">
+                <div class="right-image wow fadeInRight" data-wow-duration="1s" data-wow-delay="0.5s">
+                  <img src="assets/images/qe/phone2.png" alt="" style="z-index: 3; width: 410px;">
+                </div>
+              </div>
+            </div>
+            <br>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div id="services" class="services section">
+    <div class="container">
+      <div class="row">
+        <div class="col-lg-8 offset-lg-2">
+          <div class="section-heading  wow fadeInDown" data-wow-duration="1s" data-wow-delay="0.5s">
+            <h4>Eat with your heart</h4>
+              
+            <img src="assets/images/heading-line-dec.png" alt="">
```

Comment:
Tbh I don't really get the appeal of the weird two blue lines, looks better without imho (everywhere)